examples/grafana/grafana-ocp.yaml

```diff
@@ -0,0 +1,61 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: grafana-ocp
+  annotations:
+    "openshift.io/display-name": Grafana ocp
+    description: |
+      Grafana server with patched Prometheus datasource.
+    iconClass: icon-cogs
+    tags: "metrics,monitoring,grafana,prometheus"
+parameters:
+- description: The namespace to instantiate heapster under. Defaults to 'grafana'.
+  name: NAMESPACE
+  value: grafana
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: grafana-ocp
+    namespace: "${NAMESPACE}"
+    labels:
+      metrics-infra: grafana-ocp
+      name: grafana-ocp
+  spec:
+    selector:
+      name: grafana-ocp
+    ports:
+    - port: 3000
+      protocol: TCP
+      targetPort: 3000
+- apiVersion: v1
+  kind: ReplicationController
+  metadata:
+    name: grafana-ocp
+    namespace: "${NAMESPACE}"
+    labels:
+      metrics-infra: grafana-ocp
+      name: grafana-ocp
+  spec:
+    selector:
+      name: grafana-ocp
+    replicas: 1
+    template:
+      version: v1
+      metadata:
+        labels:
+          metrics-infra: grafana-ocp
+          name: grafana-ocp
+      spec:
+        volumes:
+        - name: data
+          emptyDir: {}
+        containers:
+        - image: "mrsiano/grafana-ocp:latest"
+          name: grafana-ocp
+          volumeMounts:
+          - name: data
+            mountPath: "/root/go/src/github.com/grafana/grafana/data"
+          command:
+          - "./bin/grafana-server"
```

Comment:
Could you please add a route to this template?  Here is an example:
https://github.com/openshift/origin/blob/master/examples/prometheus/prometheus.yaml#L52

Reply:
note:

when implementing the route, note that here the grafana server run without tls, so outside port is 80 not 443 and you do not need the tls part

Reply:
p.s.

It will be nicer to run grafana with tls ...

Reply:
for tls, you will have to edit:
https://github.com/grafana/grafana/blob/master/conf/defaults.ini#L29

you can get the key/cert using a secret, see example [1]

[1] https://github.com/openshift/origin/blob/master/examples/prometheus/prometheus.yaml#L70

Reply:
@jeremyeder @yaacov 
It dose not work, I tried to resolve that already thats why I use ox expose comand, it needs further investigation.

Reply:
If it's OK, let's handle TLS in a subsequent PR.

Reply:
> If it's OK, let's handle TLS in a subsequent PR.

Sounds right by me 👍

Reply:
+1 
So ill revert the https for now and will handle it later on by different patch.

Reply:
@jeremyeder route is added 🍰 , PTAL

Reply:
Works for me.  I am able to create dashboards now :-)